project/HaHaFi/proposal_1/README.md

```diff
@@ -0,0 +1,67 @@
+#  LSD (Line Segment Detector)
+## Introdution
+In computer vision region, detection of the line on the images in real time is an useful function, because it has a lot of downstream task, such as vanish point method base camera calibration and road marking segmentation.
+
+![](https://i.imgur.com/WD8w34b.png)
+
+reference from [2](https://www.ipol.im/pub/art/2012/gjmr-lsd/article.pdf)
+## Problem to solve
+1. The time of line detection should be as fast as possible.
```

Comment:
To make a really fast anything is very time-consuming.  Most likely it will be outside the scope of a term project.

Reply:
I'll change it into "The implementation of this project should be the linear time algorithm as the same as in paper".